pages/vi/vi-create-issues-and-pull-requests.md

```diff
@@ -32,6 +32,84 @@ You are encouraged to post as many issues as you can for improving the page as w
 
 **HINT**: While waiting for two OLE team members to approve your Pull Requests, you can continue creating more Issues and Pull Requests (upon approval, we suggest). They will all count towards your final Issue/PR total. PR approval can take time, so don't let a lengthy approval/fix process stop you from progressing in your "First Steps".
 
+## Adding HTML Games on Planet.vi
+
+In this section, you will learn how to add HTML games to Planet.vi. Follow these steps carefully, and make sure to reference the screenshots provided for better clarity.
+
+### Steps to Add HTML Games:
+
+#### **Step 1: Visit the Planet.vi Website**
+- Go to [https://planet.vi.ole.org/](https://planet.vi.ole.org/).
+- **(Screenshot of the homepage)**
+
+<img src="https://github.com/user-attachments/assets/ce061b97-9420-49c6-aa8c-e9ce97dd2708" width="800" alt="Planet.vi Homepage">
```

Comment:
I wonder where you uploaded the images? One of the potential issue I see is that [those images might be deleted after some time](https://github.com/orgs/community/discussions/77006).

Could you add them to `pages/vi/images/` instead?